docs/source/developer/contributing.rst

```diff
@@ -251,6 +251,17 @@ pass without a reply that unblocks it.
 Our expectation is that every new issue should be examined within a week of
 its creation.
 
+Triagers should label easier issues as ``good first issue`` to
+facilitate beginner contributions. A good first issue should have:
+
+* A clear, easily understood description, with screen shots and expectations that do not require much familiarity with the product
+* Links, either in the description or in comments, to documentation and source code files that are relevant to the issue
+* Recommended points of contact, either by GitHub username or on other forums (Discourse, etc) where a developer can get help
```

Comment:
Suggest replacing developer with contributor
as we often require skills beyond development such as design or accessibility knowledge

Reply:
Replaced in 29cd2ea.